node/runtime/pangolin/src/lib.rs

```diff
@@ -123,7 +124,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: sp_runtime::create_runtime_str!("Pangolin"),
 	impl_name: sp_runtime::create_runtime_str!("Pangolin"),
 	authoring_version: 0,
-	spec_version: 2_9_02_0,
+	spec_version: 29_020,
```

Comment:
@AurevoirXavier Look good for you?

Reply:
Please keep the origin style here. @AsceticBear

Reply:
I am okay with both. Fixed